css/application.css

```diff
@@ -0,0 +1,311 @@
+html,
+body,
+div,
+span,
+applet,
+object,
+iframe,
+h1,
+h2,
+h3,
+h4,
+h5,
+h6,
+p,
+blockquote,
+pre,
+a,
+abbr,
+acronym,
+address,
+big,
+cite,
+code,
+del,
+dfn,
+em,
+img,
+ins,
+kbd,
+q,
+s,
+samp,
+small,
+strike,
+strong,
+sub,
+sup,
+tt,
+var,
+b,
+u,
+i,
+center,
+dl,
+dt,
+dd,
+ol,
+ul,
+li,
+fieldset,
+form,
+label,
+legend,
+table,
+caption,
+tbody,
+tfoot,
+thead,
+tr,
+th,
+td,
+article,
+aside,
+canvas,
+details,
+embed,
+figure,
+figcaption,
+footer,
+header,
+hgroup,
+menu,
+nav,
+output,
+ruby,
+section,
+summary,
+time,
+mark,
+audio,
+video {
+  margin: 0;
+  padding: 0;
+  border: 0;
+  font: inherit;
+  font-size: 100%;
+  vertical-align: baseline;
+}
+
+/* HTML5 display-role reset for older browsers */
+article,
+aside,
+details,
+figcaption,
+figure,
+footer,
+header,
+hgroup,
+menu,
+nav,
+section {
+  display: block;
+}
+
+ol,
+ul {
+  list-style: none;
+}
+
+blockquote,
+q {
+  quotes: none;
+}
+
+blockquote::before,
+blockquote::after {
+  content: "";
+  content: none;
+}
+
+q::before,
+q::after {
+  content: "";
+  content: none;
+}
+
+table {
+  border-collapse: collapse;
+  border-spacing: 0;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  font-size: 15px;
+  line-height: 1;
+}
+
+a {
+  text-decoration: none;
+  color: #4bbcf6;
+}
+
+ul li {
+  display: inline;
+}
+
+.navbar {
+  position: absolute;
+  top: 15vh;
+  left: 12.5vw;
+  right: 12.5vw;
+  right: auto;
+  background: linear-gradient(#707070, #424242, #707070);
+  color: #fff;
+  width: 75vw;
+  margin: 0 auto;
+  border-radius: 4px;
+}
+
+.navbar ul {
+  height: 6vh;
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr 1fr 1fr 1fr 1fr 1fr 1.5fr;
+}
+
+.navbar ul li {
+  display: flex;
+  align-content: center;
+  align-items: center;
+  justify-content: center;
+  border-right: 1px solid #424242;
+  padding: 0 1em;
+}
+
+.navbar ul li:hover {
+  background: radial-gradient(#707070, #222);
+}
+
+.navbar ul li a {
+  color: #fff;
+}
+
+.navbar ul .nav-form {
+  border-right: none;
+}
+
+.navbar ul .nav-form:hover {
+  background: transparent;
+}
+
+.navbar ul .nav-form form div input {
+  width: 90%;
+  border-radius: 10px;
+  background: transparent;
+  transition-duration: 0.7s;
+}
+
+.navbar ul .nav-form form div input:focus {
+  text-decoration: none;
+}
+
+.navbar ul .nav-form form div input:hover {
+  background: #fff;
+  width: 95%;
+  transition: 20ms;
+}
+
+.jumbo {
+  font-family:
+    "Lucida Grande",
+    "Lucida Sans Unicode",
+    Helvetica,
+    Arial,
+    sans-serif;
+  background-image: url(/images/your_verse_hero.jpg);
+  background-position: bottom;
```

Comment:
According to your repository, I see that the path that you are referencing for the background image is wrong. :pray: check it

Reply:
It has been fixed..